### PR TITLE
feat(projects): Texts adjustments (#140)

### LINKED ISSUE #140: Projects texts adjustments
- [x] Update projects title
- [x] Update CETIROISE project texts
- [x] Add projects start and end dates
- [x] Update logo shown for each projects: APOCADO=OFB, CETIROISE=France relance, OFB, Parc marin d'iroise

### PR DESCRIPTION
**Pull request by ElodieENSTA**
Fixes #140 
- [x] Update projects title
- [x] Update CETIROISE project texts
- [x] Add projects start and end dates
- [x] Update logo shown for each projects: APOCADO=OFB, CETIROISE=France relance, OFB, Parc marin d'iroise

### CODE REVIEW

File: website/src/pages/Projects/styles.css [hunk 1]
```diff
@@ -1,3 +1,10 @@
 div#projects-page img {
     max-height: 80vh;
 }
+div#projects-page .container {
+    display: grid;
+    gap: 8rem;
+}
+div#projects-page figure > img.logo {
+    max-width: 200px;
+}
```
Comment (ixio) [inline website/src/pages/Projects/styles.css:10]: I'm a bit annoying with this but please be careful to always leave a newline at the end of your files. Every time you make a pull request please double-check the files view to see the github diff before asking a review. This view will show you this small red warning if you have files without an extra newline at the end. We can talk about it if you don't know why this is even a thing :wink:

**Review by ixio — CHANGES_REQUESTED** [review]
One small detail otherwise this looks good :+1:

File: website/src/pages/Projects/styles.css (hunk 1 above)
Comment (ElodieENSTA) [reply website/src/pages/Projects/styles.css]: It should be ok now.

**Review by ixio — APPROVED** [review]
LGTM

**Pull request merged**